### DISCUSSION TITLE
Cannot find module 'reflect-metadata' yarn

### DISCUSSION THREAD

**Discussion by steveyout** [adonisjs/core#4056] [Help] [answered]
i get this error ```Cannot find module 'reflect-metadata'``` while running  ```node ace configure adonis-request-throttler``` . Am using yarn as my package mannager

**Comment by jsdecena** [comment]
+1 I am also having this error with yarn
<img width="667" alt="image" src="https://github.com/adonisjs/core/assets/2285625/b33bafa1-4b4a-45ec-bfb0-9043622838cf">

**Comment by steveyout** [comment, marked as answer]
i found the solution.Try running with yarn before the command eg ```yarn node ace configure adonis-request-throttler```
